src/main/treeDataProviders/utils/analyzerUtils.ts

```diff
@@ -248,9 +251,20 @@ export class AnalyzerUtils {
                             evidence: evidences
                         } as IApplicableDetails;
                     } else {
-                        // Not applicable
+                        // Not Applicable
```

Comment:
if cve is not in `applicableCve` does not automatic determined as not applicable any more.
You should take it into considerations, when happened on CVE that is undetermined/ not covered?...

Reply:
Currently. If a CVE is not in status applicable or not applicable I'll end in line 256 condition and will not assign a value.